core/domain/state_domain.py

```diff
@@ -1449,9 +1449,6 @@ def __init__(self, data_format, translation, needs_update):
         self.translation = translation
         self.needs_update = needs_update
 
-        if data_format == self.DATA_FORMAT_HTML:
```

Comment:
I've removed this step from the object initialization step as it takes 3 secs to complete create an exp object out of models (a big exp model). Instead of cleaning a data fetched from the model, I've moved the step to update method i.e, where we are updating the HTML fields.